analyzers/src/SonarAnalyzer.CSharp/Rules/MessageTemplates/LoggingTemplatePlaceHoldersShouldBeInOrder.cs

```diff
@@ -97,7 +98,7 @@ placeholderValue switch
         {
             MemberAccessExpressionSyntax memberAccess => MatchesName(placeholderName, memberAccess.Name),
             CastExpressionSyntax cast => MatchesName(placeholderName, cast.Expression),
-            NameSyntax name => new string(placeholderName.Where(char.IsLetterOrDigit).ToArray()).Equals(name.GetName(), StringComparison.OrdinalIgnoreCase),
+            NameSyntax name => placeholderName.Equals(name.GetName(), StringComparison.OrdinalIgnoreCase) || (placeholderName.Length >= 5 && name.GetName().IndexOf(placeholderName, StringComparison.OrdinalIgnoreCase) >= 0),
```

Comment:
I'm not sure I understand where 5 is coming from.

Reply:
Removed it.